tests/integration/callbacks/test_pattern_matching.py

```diff
@@ -0,0 +1,384 @@
+from selenium.webdriver.support.select import Select
```

Comment:
Is this import used?

Reply:
No, thanks for catching that, removed in d777ae1.